setup.py

```diff
@@ -27,7 +27,7 @@ def read(fname):
       author='Parag Baxi',
       author_email='parag.baxi@gmail.com',
       description='QualysGuard(R) Qualys API Package',
-      license ='BSD-new',
+      license="Apache License (2.0)",
```

Comment:
Probably best to use BSD-new OR  Apache so we don't push an incompatible license change on  anyone by accident